packages/dso-toolkit/src/components/modal/modal.mixins.scss

```diff
@@ -191,13 +229,46 @@
   }
 }
 
+@mixin fullscreen($target) {
+  // always declare target
+  @if $target == "core" {
```

Comment:
Tsja ik begrijp het.. Maar eigenlijk is dit niet de bedoeling. De package in `packages/dso-toolkit` heeft geen weet van een implementatie zoals HTML/CSS of core. De specifieke styling voor deze implementaties doen we direct in de betreffende codebase.

Dit komt vanwege de dependency tree:

`@dso-toolkit/core > dso-toolkit`: De package `dso-toolkit` heeft geen weet van `@dso-toolkit/core`, andersom wel. Het is belangrijk om die dependency ook in je code na te streven.

Reply:
Zie hierboven :)

Reply:
🤘